content/docs/03-ai-sdk-core/36-transcription.mdx

```diff
@@ -151,5 +151,10 @@ try {
 | [OpenAI](/providers/ai-sdk-providers/openai#transcription-models)         | `gpt-4o-mini-transcribe` |
 | [ElevenLabs](/providers/ai-sdk-providers/elevenlabs#transcription-models) | `scribe_v1`              |
 | [ElevenLabs](/providers/ai-sdk-providers/elevenlabs#transcription-models) | `scribe_v1_experimental` |
+| [Deepgram](/providers/ai-sdk-providers/deepgram#transcription-models)     | `base` (+ variants)      |
```

Comment:
what do variants mean in this context?

Reply:
https://github.com/vercel/ai/blob/f8492461a4e57f5a4ea43b55c6f7da87aa749f85/packages/deepgram/src/deepgram-transcription-settings.ts

Reply:
maybe we could add a link to: https://developers.deepgram.com/docs/models-languages-overview